pennylane/labs/dla/cartan_subalgebra.py

```diff
@@ -0,0 +1,269 @@
+        :math:`[\mathfrak{k}, \mathfrak{k}] \subeq \mathfrak{k}`, :math:`[\mathfrak{k}, \mathfrak{m}] \subeq \mathfrak{m}` and :math:`[\mathfrak{m}, \mathfrak{m}] \subeq \mathfrak{k}`,
+        which we can check using the helper function :func:`~check_cartan_decomp`.
+
+        >>> from pennylane.labs.dla import even_odd_involution, check_cartan_decomp
+        >>> k, m = cartan_decomp(g, even_odd_involution)
+        >>> check_cartan_decomp(k, m) # check commutation relations to be valid Cartan decomposition
+        True
+
+        Our life is easier when we use a canonical ordering of the operators. This is why we re-define ``g`` with the new ordering in terms of operators in ``k`` first, and then
+        all remaining operators from ``m``.
+
+        >>> g = np.vstack([k, m]) # re-order g to separate k and m operators
+        >>> adj = structure_constants_dense(g) # compute adjoint representation of g
+
+        Finally, we can compute a Cartan subalgebra :math:`\mathfrak{h}`, a maximal Abelian subalgebra of :math:`\mathfrak{m}`.
+
+        >>> newg, k, mtilde, h, new_adj = cartan_subalgebra(g, k, m, adj, start_idx=3)
+
+        The new DLA ``newg`` is just the concatenation of ``k``, ``mtilde``, ``h``. Each component is returned in the original input format.
+        Here we obtain collections of :math:`8\times 8` matrices (``numpy`` arrays), as this is what we started from.
+
+        >>> newg.shape, k.shape, mtilde.shape, h.shape, new_adj.shape
+        ((15, 8, 8), (6, 8, 8), (6, 8, 8), (3, 8, 8), (15, 15, 15))
+
+        We can also let the function return what we call adjoint representation vectors.
+
+        >>> np_newg, np_k, np_mtilde, np_h, new_adj = cartan_subalgebra(g, k, m, adj, start_idx=3, return_adjvec=True)
+        >>> np_newg.shape, np_k.shape, np_mtilde.shape, np_h.shape, new_adj.shape
+        ((15, 15), (6, 15), (6, 15), (3, 15), (15, 15, 15))
+
+        These are dense vector representations of dimension :math:`|\mathfrak{g}|`, in which each entry corresponds to the respective operator in :math:`\mathfrak{g}`.
+        Given an adjoint representation vector :math:`v`, we can reconstruct the respective operator by simply computing :math:`\hat{O}_v = \sum_i v_i g_i`, where
+        :math:`g_i \in \mathfrak{g}` (hence the need for a canonical ordering).
+
+        We provide a convenience function :func:`~adjvec_to_op` that works with both ``g`` represented as dense matrices or PL operators.
+        Because we used dense matrices in this example, we transform the operators back to PennyLane operators using :func:`~pauli_decompose`.
+
+        >>> from pennylane.labs.dla import adjvec_to_op
+        >>> h = adjvec_to_op(np_h, g)
+        >>> h_op = [qml.pauli_decompose(op).pauli_rep for op in h]
+        >>> h_op
+        [0.35355339059327384 * Y(1) @ Y(2),
+         0.3535533905932738 * Y(0) @ Y(2),
+         -0.3535533905932738 * Y(0) @ Y(1)]
```

Comment:
My output seems different. Can you please double-check?

Reply:
Yep I think it is because we changed the default behavior concerning normalization 👍